packages/shared/src/hooks/input/useMarkdownInput.ts

```diff
@@ -97,6 +98,7 @@ export const useMarkdownInput = ({
   initialContent = '',
   enabledCommand = {},
 }: UseMarkdownInputProps): UseMarkdownInput => {
+  const dirtyRef = useRef(false);
```

Comment:
Not sure, but would a usestate not also work?
Im not sure which one performs betetr tbh